Model/Cron.php

```diff
@@ -1986,7 +1986,11 @@ protected function _createInvoice()
         } else {
             $this->_adyenLogger->addAdyenNotificationCronjob
             (
-                'It is not possible to create invoice for this order'
+                'It is not possible to create invoice for this order',
+                [
```

Comment:
could you pls also log the orderId or orderNumber to make it easier to find the order?